make/targets/golang/verify-update.mk

```
@@ -3,11 +3,12 @@ include $(addprefix $(dir $(lastword $(MAKEFILE_LIST))), \
 )
 
 go_files_count :=$(words $(GO_FILES))
+chunk_size :=1000
 
 verify-gofmt:
 	$(info Running `$(GOFMT) $(GOFMT_FLAGS)` on $(go_files_count) file(s).)
 	@TMP=$$( mktemp ); \
-	$(GOFMT) $(GOFMT_FLAGS) $(GO_FILES) | tee $${TMP}; \
+	echo $(GO_FILES) | xargs -n $(chunk_size) | xargs $(GOFMT) $(GOFMT_FLAGS) | tee $${TMP}; \
```

I'm not sure the chunking is happening the way you expect with the extra pipe in place. I suspect you meant
```suggestion
	echo $(GO_FILES) | xargs -n $(chunk_size) $(GOFMT) $(GOFMT_FLAGS) | tee $${TMP}; \
```

...which, bonus, saves a subprocess :)